bundled/tool/lsp_server.py

```diff
@@ -202,6 +202,7 @@ def _clear_diagnostics(document: workspace.Document) -> None:
 
 
 def _linting_helper(document: workspace.Document) -> None:
+    # pylint: disable=global-variable-not-assigned
```

Comment:
You can remove this. add the error code to https://github.com/microsoft/vscode-mypy/blob/main/.flake8